examples/demo-cilium-egress/scripts/teardown.sh

```
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+# Teardown. By default removes only the demo resources (keeps the cluster).
+# Pass --all to also delete the GKE cluster.
+set -uo pipefail
+source "$(dirname "${BASH_SOURCE[0]}")/../env.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./../env.sh was not specified as input (see shellcheck -x).

(SC1091)